rero_ils/modules/documents/templates/rero_ils/detailed_view_documents.html

```diff
@@ -193,5 +193,18 @@ <h5 class="d-inline-block">{{ _('Export Formats') }}:</h5>
 {{ super() }}
 {% assets "rero_ils_documents_detailed_js" %}
 <script src="{{ ASSET_URL }}"></script>
+
+<script type="text/javascript">
+$( window ).load(function() {
+    /* Store state of view tab */
+    $('a[data-toggle="tab"]').on('click', function(e) {
+      localStorage.setItem('activeTabDoc', $(this).attr('href'));
```

Comment:
We need a namespace here: something like `reroILSActiveTabDetailedDocument` or similar...